### PR TITLE
OP-1188 | Delete Inventory

### PR DESCRIPTION
**Pull request by ArnaudFonzam**
See OP-1188.

### CODE REVIEW

File: src/main/java/org/isf/medicalinventory/service/MedicalInventoryIoOperation.java [hunk 1]
```diff
@@ -22,12 +22,16 @@
 package org.isf.medicalinventory.service;
 
 import java.time.LocalDateTime;
+import java.util.ArrayList;
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/medicalinventory/service/MedicalInventoryIoOperation.java]: Unnecessary import

File: src/main/java/org/isf/medicalinventory/service/MedicalInventoryIoOperation.java [hunk 2]
```diff
@@ -22,12 +22,16 @@
...
 import java.util.List;
 import java.util.Optional;
 
+import org.isf.generaldata.MessageBundle;
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/medicalinventory/service/MedicalInventoryIoOperation.java]: Unnecessary import

File: src/main/java/org/isf/medicalinventory/service/MedicalInventoryIoOperation.java [hunk 3]
```diff
@@ -22,12 +22,16 @@
...
+import org.isf.medicalinventory.model.InventoryStatus;
 import org.isf.medicalinventory.model.MedicalInventory;
 import org.isf.utils.db.TranslateOHServiceException;
 import org.isf.utils.exception.OHServiceException;
+import org.isf.utils.exception.model.OHExceptionMessage;
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/medicalinventory/service/MedicalInventoryIoOperation.java]: Unnecessary import

**Comment by ArnaudFonzam** [comment]
Hello I have close this PR because I used previously the Intesys repos And they need to close all repos conserning oh. I have create new PR by using my repos [OP-1188](https://github.com/informatici/openhospital-core/pull/1400)

[OP-1188]: https://openhospital.atlassian.net/browse/OP-1188?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

**Pull request closed without merging**